build.rs

```diff
@@ -1,9 +0,0 @@
-extern crate embed_resource;
```

Comment:
About this file - without it, executable will not have an icon on Windows.

Reply:
but it was never triggered... 
build.rs must be set in Cargo.toml - I will rereview that later on

Reply:
fixed!